.github/workflows/test-application.yaml

```diff
@@ -204,6 +204,11 @@ jobs:
                   ini-values: memory_limit=-1
                   coverage: none
 
+            - name: Remove Lint Tools
+              # This tools are not required for running tests and are so removed to improve dependency resolving and
+              # testing lowest versions
+              run: composer remove "*php-cs-fixer*" "*phpstan*" --dev --no-update
+
```

Comment:
This improves the CI dependency resolving as this dependencies are not required for tests and so composer has less todo.